Frameworks/CoreText/CTLine.mm

```diff
@@ -278,21 +303,24 @@ void CTLineDraw(CTLineRef lineRef, CGContextRef ctx) {
  @Status Interoperable
 */
 CFIndex CTLineGetGlyphCount(CTLineRef line) {
-    return line ? static_cast<_CTLine*>(line)->_glyphCount : 0;
+    RETURN_RESULT_IF_NULL(line, 0);
+    return line->_glyphCount;
```

Comment:
inline functions in struct are pretty awesome ;P

---
In reply to: [112742078](https://github.com/Microsoft/WinObjC/pull/2586#discussion_r112742078) [](ancestors = 112742078)